fuzzy/docker/level1/Dockerfile.libmath

```diff
@@ -0,0 +1,17 @@
+FROM verificarlo/verificarlo:latest
+
+RUN apt-get update &&\
+    apt-get install -y make build-essential wget
```

Comment:
I think `wget` and `build-essential` are in the container I mentioned using as a base, above. In which case, these could be removed.

Reply:
Yes, I changed docker base.